### ISSUE TITLE
addKnownDataPoints.js is not working

### ISSUE THREAD

**Issue by enricoar** [ankitrohatgi/WebPlotDigitizer-Examples#3]
Actually it should be pretty simple: just run script and at least one should see a couple of more points but it does not seem to work.

I notice the examples are 5 years old, so maybe they are not up-to-date?

**Comment by ankitrohatgi** [comment]
Yup, these examples are super out-of-date. I'll fix this :soon:

**Comment by ankitrohatgi** [comment]
Try this: https://github.com/ankitrohatgi/WebPlotDigitizer/blob/master/script_examples/addKnownDataPoints.js

**Comment by enricoar** [comment]
Thanks a lot for the quick replay!  The script does work for the x-y plot but for example not for the map with scale bar. I'll try to find out. Thanks for your help

**Comment by ankitrohatgi** [comment]
Hmm, I tried it for a map with scale bar and it worked. There are a couple of things to watch out for:

1. Make sure you have selected the dataset that you are modifying in the tree on the left. The script just inserts points into whatever the current active dataset is.
2. Make sure the Axes assigned to the dataset is correct (Map).
3. The (0, 0) for map with scale bar is bottom left and not top left like an image.

**Comment by enricoar** [comment]
Thanks a lot. Sorry I haven't looked at this particular issue for some 
time. Soon there will be a  publication in archaeology citing your code.

Going back to the issue,  which I tested again. I had already taken into 
account your three points below.  I checked the points via "show data" 
and they are always in (0,0) no matter what. The point is that when 
choosing the scale bar dataPx always comes out to be (0,0) , I checked 
it with a  wpd.messagePopup.show

Thanks
